tests/baselines/reference/checkJsdocSatisfiesTag11.errors.txt

```diff
@@ -23,8 +22,6 @@
     
     /**
      * @satisfies {number}

```

Comment:
as shown by the new `tests/cases/conformance/jsdoc/checkJsdocSatisfiesTag16.ts` both are still checked - but they are no longer treated as duplicates. I think this is fine but maybe @sandersn would have a different opinion

Reply:
It's correct to drop the error, but the first tag shouldn't even be checked--the comment on ownsJSDocTag says that satisfies tags not on parentheses don't apply to anything. I think the real bug is that type tags shouldn't apply to the parenthesized expression in this case either, but fails because `@type` usage is ambiguous.